lib/PerlIO.pm

```diff
@@ -247,10 +358,10 @@ Some custom layers come with the Perl distribution.
 
 =item :encoding
 
-Use C<:encoding(ENCODING)> to transparently do character set and encoding
-transformations, for example from Shift-JIS to Unicode.  Note that an
-C<:encoding> also enables C<:utf8>.  See L<PerlIO::encoding> for more
-information.
+Use C<:encoding(ENCODING)> either in open() or binmode() to install
```

Comment:
The unnecessary phrase from `either in open() or binmode() to install a layer` doesn't need to be reintroduced.

Reply:
This has been reverted.